lib/tasks/trigger_ea_events.rake

```diff
@@ -5,6 +5,8 @@ require 'rake'
 namespace :trigger do
   desc 'Send an event if early allocation changed OOB (due to e.g. time passing)'
   task early_allocation_events: :environment do
+    next if ENABLE_EVENT_BASED_HANDOVER_CALCULATION
```

Comment:
Should this be return?

Reply:
No, 'cos we're in a block, not a method 🖕🏾